modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -130,17 +130,27 @@ Do not apply any expiration rules. Delete all versions after 30 days.
 'versions_retention_obligation' => '30, 30',
 ----
 
-== Show the Author of a Version
+== Restoring of a Version
 
-You can enable and disable to show the author of a version in the versions history via an entry in config.php. For details see: xref:configuration/server/config_sample_php_parameters.adoc#save-and-display-the-author-of-each-version-of-uploaded-and-edited-files[Save and display the author of each version of uploaded and edited files]. This is helpful to keep track of who changed a version. 
+In the WebUI in the details view, there is a restore button next to the versions. It can be used to restore file to its past version. Restore does not delete old version, instead newer version is created. 
```

Comment:
```suggestion
In the WebUI in the details view, there is a restore button next to the versions. It can be used to restore the file to a past version. Restore does not delete the old version, instead a new current version is created. 
```

Does "a new current version" make sense? It does to me, but maybe other people have better words for it?